### ISSUE TITLE
[tesserini] migliorare usabilita

### ISSUE THREAD

**Issue by AlfioEmanueleFresta** [CroceRossaItaliana/jorvik#283]
Spesso la gente segnala che lo scaricamento dei tesserini non funziona. Ogni singola volta che e' successo questo mi e' sembrato di capire che la problematica sia la stessa: la gente ignora ogni tipo di istruzione, seleziona i tesserini e clicca "scarica" (*non* "processa e scarica"). In pratica seleziona per lo scaricamento un elenco di tesserini che contiene (spesso, ma non sempre, unicamente) tesserini NON processati.

Ora, i tesserini che non sono ancora stati processati come approvati, non hanno un codice di tesserino. Quindi lo scaricamento fallisce silenziosamente.

Bisogna fare in modo che se nella selezione c'e' almeno un tesserino non processato, nella pagina di scaricamento dia un elenco in rosso *ben visibile* che dica qualcosa come:

**42 dei tesserini selezionati _non verranno scaricati_.**
Hai selezionato delle richieste per dei tesserini che non sono ancora state processate (accettate). Questi non verranno scaricati. Processa le richieste in attesa prima di scaricarne il tesserino.

**Comment by yakky** [comment]
@AlfioEmanueleFresta qui non converrebbe inibire il solo scaricare se ci sono tesserini non processati? Gli utenti non leggeranno comunuque :)

**Comment by luca-dex** [comment]
Io sarei dell'idea di bloccarti a monte e non far partire neanche lo scaricamento. Se selezioni dei tesserini che ancora non sono stati processati ti blocco l'operazione e ti dico che ci sono dei tesserini che non sono processati, onde evitare che ne hai scaricati un po' pensi di averli scaricati tutti e ti dimentichi di quelli che sono rimasti indietro, no?

**Comment by AlfioEmanueleFresta** [comment]
Concordo con @luca-dex. Meglio dare un errore esplicativo oltre al bloccare semplicemente lo scaricamento. Es.: "Non puoi scaricare questi tesserini perche' la selezione include dei tesserini che non sono stati ancora processati. Processa tutti i tesserini che intendi stampare e prova di nuovo."

**Comment by yakky** [comment]
PR deployata su luke
